packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html

```diff
@@ -2,32 +2,40 @@
   <div class="nav-down">
     <div class="intranet-header">
       <div class="d-flex align-items-center flex-nowrap">
-        <div class="bg-logo d-sm-block">
+        <div id="logo" class="bg-logo d-sm-block">
           <a [href]="this.logoUrl" class="navbar-brand header-logo">
             <span class="visually-hidden">{{ getPostLogoText() }}</span>
           </a>
         </div>
-        <div class="title flex-shrink-1 px-4">
+        <div
+          class="title px-4"
+          id="title"
+          [ngClass]="{ 'flex-shrink-1': !condenseHeader, 'flex-shrink-0': condenseHeader }"
+        >
           <h1 class="m-0 font-weight-bold">{{ siteTitle }}</h1>
         </div>
-        <div *ngIf="showIntranetSearch" class="col search-textbox d-none d-md-block">
-          <form [action]="searchUrl">
-            <div class="input-group mx-auto">
-              <input
-                type="text"
-                name="k"
-                class="form-control"
-                placeholder="{{ getPlaceholderSearchIntranet() }}"
-              />
-              <div class="input-group-append d-flex align-items-center">
-                <button class="btn pr-4" type="submit">
-                  <span><em class="pi pi-2069"></em></span>
-                </button>
-              </div>
-            </div>
-          </form>
+
+        <nav
+          *ngIf="condenseHeader && hasNavbar"
+          aria-label="navigation"
```

Comment:
This causes screenreaders to announce "navigation navigation" as the `<nav>` already prompts a (localized) announcement of navigation. When adding this statically, there is also no chance to localize it.

```suggestion
```